### ISSUE TITLE
(Suggestion) Bring more colorful wallpapers

### ISSUE THREAD

**Issue by RandomPenguin1778** [trenta-io/trentaos-core#7]
Dear Trenta OS Team ,

I Like Trenta OS And it has beautiful wallpapers . But can you bring more colorful and more beautiful wallpapers ?

Sincerely,
RandomPenguin

**Comment by agam778** [comment]
If you can make some, it will be very good.

But we will try to bring more wallpapers.

**Comment by RandomPenguin1778** [comment]
Hi ,
I added a wallpaper , please check the pull requests at trenta.io/trenta-OS-wallapers

**Comment by GyGit79** [comment]
Yeah I am in love with the wallpapers Trenta OS has. I also expect random wallpapers but expressing as a story.

**Comment by GyGit79** [comment]
Well each taken picture and graphic has its own story.
